tool/release/notes/Version.kt

```diff
@@ -21,65 +21,76 @@
 
 package com.vaticle.dependencies.tool.release.notes
 
-data class Version(val major: Int, val minor: Int, val patch: Int, val alpha: Int?): Comparable<Version> {
+data class Version(val major: Int, val minor: Int, val patch: Int, val rc: Int?, val alpha: Int?): Comparable<Version> {
```

Comment:
_Technically_ we should have a proper prerelease type for this. Currently it's possible to have both alpha and rc here. That is out of scope for this PR though.